tools/dm/monitor.md

```diff
@@ -71,4 +71,6 @@ The following metrics show only when `task-mode` is in the `incremental` or `all
 | cost of binlog event transform | The time it takes Syncer to parse and transform the binlog into SQL statements (in seconds) | N/A |
 | total sqls jobs | The number of newly added jobs per unit of time | N/A |
 | finished sqls jobs | The number of finished jobs per unit of time | N/A |
-| execution latency | The time it takes Syncer to execute the transaction to the downstream (in seconds) | N/A |
\ No newline at end of file
+| execution latency | The time it takes Syncer to execute the transaction to the downstream (in seconds) | N/A |
+| unsynced tables | The number of tables that have not received the sharding DDL statement in the current subtask | N/A |
+| shard lock resolving | Whether the current subtask is waiting for the sharding DDL statement to be replicated; a value greater than 0 indicates that it is waiting for the DDL replication | N/A |
```

Comment:
how about `is waiting for the shard DDL lock to be resolved`?
It's scope is smaller than `is waiting for the sharding DDL statement to be replicated`